src/main/java/com/entropyteam/entropay/auth/AuthConstants.java

```diff
@@ -0,0 +1,6 @@
+package com.entropyteam.entropay.auth;
+
+public class AuthConstants {
```

Comment:
please make this class final and add a private constructor. Or move the constant to a different component class.